js/app.js

```diff
@@ -10,17 +10,12 @@ function Project(rawDataObj){
 }
 
 Project.prototype.toHtml = function(){
-  var $newProject = $('.template').clone();
-  $newProject.removeClass('template');
+  var template = $('#project-template').html();
+  var templateRender = Handlebars.compile(template);
+  this.daysAgo = parseInt((new Date() - new Date(this.createdOn))/60/60/24/1000);
+  this.createStatus = this.createdOn ? 'created ${this.daysAgo} days ago' : '(draft)';
```

Comment:
You probably noticed that, if you changed `<p>{{createdOn}}</p>` (`line 15` in `index.html`)
to `<p>{{createStatus}}</p>`, it won't show the correct result because over here on `line 16`,
you're using `'` instead of ```. 
`'created ${this.daysAgo} days ago'` should be ``created ${this.daysAgo} days ago``